README.markdown

```diff
@@ -424,6 +424,18 @@ After the v0.8.6 release of node, nvm tries to install from binary packages. But
 
 If setting the `default` alias does not establish the node version in new shells (i.e. `nvm current` yields `system`), ensure that the system's node PATH is set before the `nvm.sh` source line in your shell profile (see [#658](https://github.com/creationix/nvm/issues/658))
 
+## Mac OS "troubleshooting"
+
+**nvm node version not found in vim shell**
+
+If you set node version to a version other than your system node version `nvm use 6.2.1` and open vim and run `:!node -v` you should see `v6.2.1` if you see your system version `v0.12.7`. You need to run: 
```

Comment:
Somebody's system version may not be `v0.12.7`, and they may not even have a system version - can we make this language a bit more generic?